src/databricks/labs/ucx/cli.py

```diff
@@ -582,6 +582,14 @@ def assign_metastore(
     logger.info(f"Account ID: {a.config.account_id}")
     ctx = AccountContext(a)
     ctx.account_metastores.assign_metastore(ctx.prompts, workspace_id, metastore_id, default_catalog)
+    ctx.catalog_schema.create_ucx_catalog(ctx.prompts)
```

Comment:
@nfx : Note that prompts are on the `AccountContext` instead of a argument to the cli command. Do you know why?